<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettings</class>
 <widget class="QDialog" name="CSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="m_networkCom">
     <property name="toolTip">
      <string>Show or hide the visualization of network communications. </string>
     </property>
     <property name="toolTipDuration">
      <number>-5</number>
     </property>
     <property name="text">
      <string>Show network communications</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_search">
     <property name="toolTip">
      <string>To update playlists with invalid entries, use the server features if they exist. The update can be faster but lead to errors.</string>
     </property>
     <property name="text">
      <string>Use server search for playlist checking</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_eventOnly">
     <property name="toolTip">
      <string>May reduce the network trafic, but also create some operating erreors.</string>
     </property>
     <property name="text">
      <string>UPnP events only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_playlists">
     <property name="toolTip">
      <string>Don't use playlists capabilities of renderer. Use only basic UPnP functions.</string>
     </property>
     <property name="text">
      <string>Don't use playlists for renderers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_cloudServers">
     <property name="toolTip">
      <string>Don't use playlists capabilities of renderer. Use only basic UPnP functions.</string>
     </property>
     <property name="text">
      <string>Show cloud servers (AIVCtrl must be restarted)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Icon size: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_iconSize">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_size">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>32</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_reset">
       <property name="toolTip">
        <string>Restore the default seetings</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
